src/ddspdrum/module.py

```diff
@@ -45,6 +44,9 @@ def __repr__(self):
             self.__class__, repr(self.sample_rate), repr(self.parameters)
         )
 
+    def seconds_to_samples(self, seconds: float) -> int:
+        return round(int(seconds * self.sample_rate))
+
```

Comment:
Controversial, again. This looks like the best place for this function.

It didn't make sense to put it in utils, it's tied to a sample rate, which is always carried in the object.

Otherwise you get a call like `seconds_to_samples(seconds_val, self.sample_rate)` which doesn't save you much space.

Reply:
I'm fine with this